var/spack/repos/builtin/packages/chaparral/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Chaparral(CMakePackage):
+    """Radiation view factor library"""
```

Comment:
The repo description mention this is a fork of an equally named project from Sandia, can you clarify that a bit? I am trying to understand if we need to reserve the "pristine" name for the original software. Thanks!

Reply:
That works perfectly. Thanks!